src/sage/rings/polynomial/multi_polynomial_libsingular.pyx

```diff
@@ -5744,6 +5744,78 @@ cdef class MPolynomial_libsingular(MPolynomial_libsingular_base):
         else:
             return self * self.denominator()
 
+    def in_subalgebra(self, J, algorithm=None):
+        """
+        Return whether this polynomial is contained in the subalgebra
+        generated by ``J``
+
+        INPUT:
+
+        - ``J`` -- iterable of elements of the parent polynomial ring
+
```

Comment:
"iterable of elememts of ..." is precise, but I think just "elements of ..." is simple and easy. Just a suggestion.

Reply:
I'll change it to "list of elements ..."